### PR TITLE
comma cleanup in docs

### PR DESCRIPTION
**Pull request by landonwilkins**
test plan:
* you like the comma changes

### CODE REVIEW

File: website/source/docs/apps/index.html.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ sidebar_current: "docs-apps"
 description: |-
   The application type in the Appfile
   is the most important piece of information that tells Otto how to function.
-  Otto uses this information to know how to develop, build, and deploy
```
Comment (sethvargo) [inline website/source/docs/apps/index.html.md]: This is an oxford comma and is preferred over the non-oxford version, especially for non-native speakers. I am against removing it, and it's grammatically correct.

Comment (sethvargo) [inline website/source/docs/apps/index.html.md]: - In serious: https://thewritepractice.com/why-you-need-to-be-using-oxford-commas/
- In fun: http://www.buzzfeed.com/adamdavis/the-oxford-comma-is-extremely-important-and-everyone-should#.igWEQLE81

Comment (landonwilkins) [inline website/source/docs/apps/index.html.md]: kk :+1:

Comment (landonwilkins) [inline website/source/docs/apps/index.html.md]: LOLOL can dig, thanks for the links

**Pull request closed without merging**